Menu/MenuBuilder.php

```diff
@@ -119,7 +119,7 @@ public function createSidebarMenu()
                     }
 
                     $label = $admin->getLabel();
-                    $options = $admin->generateMenuUrl('list');
+                    $options = $admin->generateMenuUrl('list', $item['route_params'], $item['route_absolute']);
```

Comment:
Sorry, I do not understand your sentence, could you clarify? If `route_params` is not necessary, why do you add it? Because it does not hurt? Or is there another reason behind this?